lib/rules/select-name.json

```diff
@@ -0,0 +1,25 @@
+{
+	"id": "select-name",
+	"selector": "select",
+	"tags": [
+		"cat.forms",
+		"wcag2a",
+		"wcag412",
+		"wcag131",
+		"section508",
+		"section508.22.n"
+	],
+	"metadata": {
+		"description": "Ensures select element has an accessible name",
+		"help": "Select element must have and accessible name"
+	},
+	"all": [],
+	"any": [
```

Comment:
I think these should have the role-none / role-presentation checks on them too. I know that' missing from the label rule. I've opened an issue for that.